jlogan03/tm4c129-launchpad
# Update deps, add build, and add dependabot

* Update deps to latest versions
* Add github actions config to build each crate
* Add dependabot configuration to check each crate for staleness
